main.js

```diff
@@ -1,38 +1,80 @@
-// fetch data
 const API_URL = "https://gist.githubusercontent.com/maratgaip/44060c688fcf5f2b7b3985a6d15fdb1d/raw/e93c3dce0826d08c8c6e779cb5e6d9512c8fdced/restaurant-menu.json";
 
 const dishImage = document.querySelector("img");
 const dishName = document.querySelector(".dish-name");
 const dishPrice = document.querySelector(".dish-price");
 const dishInfo = document.querySelector(".dish-info");
 const mainMenu = document.querySelector("main");
+const searchInput = document.querySelector(".search-input");
+const searchButton = document.querySelector(".search-button");
```

Comment:
You can delete `searchButton` variable, since we don't need it anymore.

Reply:
Removed.